# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome24/gedit-plugins/diff-hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- gedit-plugins-2.3.5/plugins/diff/diff.c.orig	2003-12-02 13:05:30.000000000 +0100
+++ gedit-plugins-2.3.5/plugins/diff/diff.c	2003-12-02 13:05:33.000000000 +0100
@@ -959,7 +959,7 @@
 	
 	temp = gedit_plugin_locate_program (DIFF_PROGRAM_NAME,
 					    plugin_name, 
-					    parent);
+					    parent, NULL, NULL);
 
 	if (temp != NULL)
 	{
